articles/integrate-tailwindcss-into-flask/index.md

```diff
@@ -98,9 +98,21 @@ Once the server is running head over to
 
 ## Installing TailwindCss
 
-Now that our server is up and running, let's install Tailwindcss. To install tailwind we can use either yarn or npm (depending on which one you have on your system). We navigate to the `static` folder we created earlier from the terminal using the `cd` command, once there we'll install TailwindCss in there, as follows:
+Now that our server is up and running, let's install Tailwindcss. To install tailwind we can use either yarn or npm (depending on which one you have on your system). We navigate to the `static` folder we created earlier from the terminal using the `cd` command, once there we'll use the `init` command to setup a new project to install TailwindCss in there, as follows:
```

Comment:
It would flow better as _use the init command to setup a new node.js environment, which will allow us to install TailwindCss using npm or yarn as follows:. _

Reply:
Hey thanks for the update!, I just added that